internal/service/advancedcluster/model_advanced_cluster.go

```diff
@@ -248,6 +252,11 @@ func SchemaAdvancedConfig() *schema.Schema {
 					Optional: true,
 					Computed: true,
 				},
+				"change_stream_options_pre_and_post_images_expire_after_seconds": {
+					Type:     schema.TypeInt,
+					Optional: true,
+					Default:  -1,
```

Comment:
What is the story behind the -1 default? 😅

Reply:
Basically it's the value that turns off this option